packages/ui-kit/src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -0,0 +1,95 @@
+import { Button } from '@mui/base/Button'
+import { Popper } from '@mui/base/Popper'
+import { useAutocomplete, UseAutocompleteProps } from '@mui/base/useAutocomplete'
+import { unstable_useForkRef as useForkRef } from '@mui/utils'
+import * as React from 'react'
+import classnames from 'classnames'
+import componentStyles from './Autocomplete.module.scss'
+
+// There is an issue with MUI icons: https://stackoverflow.com/a/71806305/311327
+// import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown'
+// import ClearIcon from '@mui/icons-material/Clear'
+
+// See full Autocomplete example here: https://mui.com/base-ui/react-autocomplete/#introduction
+export const Autocomplete = React.forwardRef(function Autocomplete(
+  //   props: UseAutocompleteProps<(typeof top100Films)[number], false, false, false>,
+  props: UseAutocompleteProps<any[number], false, false, false>,
+  ref: React.ForwardedRef<HTMLDivElement>
+) {
+  const { disableClearable = false, disabled = false, readOnly = false, ...other } = props
+
+  const {
+    getRootProps,
+    getInputProps,
+    getPopupIndicatorProps,
+    getClearProps,
+    getListboxProps,
+    getOptionProps,
+    dirty,
+    id,
+    popupOpen,
+    focused,
+    anchorEl,
+    setAnchorEl,
+    groupedOptions
+  } = useAutocomplete({
+    ...props,
+    componentName: 'BaseAutocompleteIntroduction'
+  })
+
+  const hasClearIcon = !disableClearable && !disabled && dirty && !readOnly
+
+  const rootRef = useForkRef(ref, setAnchorEl)
```

Comment:
I wonder if we can use our `useCombinedRefs` hook or a simple `useRef` here just to get rid of `@mui/utils`

Reply:
That's a good question. I didn't check their hook. We may consider such a replacement. Just need to investigate.

Reply:
So, we decided to use our `useCombinedRefs` instead of `useForkRef` from `@mui/utils`. We can remove `@mui/utils` from the deps list then.